src/AnnotationThread.js

```diff
@@ -213,46 +206,64 @@ class AnnotationThread extends EventEmitter {
      * @param {string} message - Text of annotation to save
      * @return {Promise} - Annotation create promise
      */
-    saveAnnotation = (type, message) => {
+    save(type, message) {
```

Comment:
As mentioned above about arrow functions. How come you moved away from them?

Reply:
So weirdly enough it wasn't actually binding in a lot of these cases? So I moved to manually binding the methods as needed.